### PR TITLE
Add getting started in README

### PR DESCRIPTION
**Pull request by nak3**
This patch adds Getting started guide to run net-gateway-api with Knative.

Also, it includes https://github.com/knative-sandbox/net-gateway-api/pull/168.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,86 @@
...
+
+```bash
+kubectl apply -f https://github.com/knative/serving/releases/latest/download/serving-crds.yaml
+kubectl apply -f https://github.com/knative/serving/releases/latest/download/serving-core.yaml
+```
+
+- Install net-gateway-api
+
+```
+ko resolve -f config/ | kubectl apply -f -
```
Comment (markusthoemmes) [inline README.md:29]:
```suggestion
ko apply -f config/
```

Comment (nak3) [reply README.md]: Thank you! Updated.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**